LICENSE.md

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2019 Project ROSS Contributors
```

Comment:
We need to update the year in our files.
In the license's case, I think we should keep the file in both repos, since github also detects and informs the license. Additionally, I don't think we are going to have many modifications to these files, so it is not a problem to keep them updated.

Reply:
Done.